scripts/devdeps.py

```diff
@@ -67,7 +71,9 @@ def depend_check(deps_name, *args):
         'pytest_cov',
         'pytest_selenium',
         'mock',
-        'websocket'
+        'websocket',
```

Comment:
It actually maps to `websocket-client` which I do have installed, so maybe it is still used in the test. Let's keep it for now